qiita_pet/templates/study_description_templates/study_information_tab.html

```diff
@@ -5,4 +7,48 @@
   <b>Principal investigator:</b> {% raw principal_investigator %} <br/>
   <b>Samples:</b> {{number_samples_promised}}/{{number_samples_collected}}<br/>
   <b>Metadata:</b> {{metadata_complete}} <br/>
+  
```

Comment:
Could you add a couple of `<br/>` or a `<hr>`, just to have a visual division between the top and the bottom?